src/internal/internalAPIs.ts

```diff
@@ -9,6 +9,13 @@ GlobalVars.handlers["fullScreenChange"] = handleFullScreenChange;
 GlobalVars.handlers["backButtonPress"] = handleBackButtonPress;
 GlobalVars.handlers["beforeUnload"] = handleBeforeUnload;
 GlobalVars.handlers["changeSettings"] = handleChangeSettings;
+GlobalVars.handlers["getConversationId"] = handleConversationId;
```

Comment:
Wouldn't a more appropriate name for this be `conversationCreated` or something like that? I was pretty confused when I first saw this code and had to go read the contents of the handler to understand what the purpose of this is.